concrete/src/Application/Service/UserInterface.php

```diff
@@ -182,10 +181,17 @@ public function getToolbarLogoSRC()
             $src = Config::get('concrete.white_label.logo');
         }
         if (!$src) {
-            $filename = 'logo.png';
+            $filename = 'logo';
+            if (file_exists(DIR_APPLICATION . '/' . DIRNAME_IMAGES . '/' . $filename . '.svg')) {
+                $src = REL_DIR_APPLICATION . '/' . DIRNAME_IMAGES . '/' . $filename . '.svg';
+            } elseif (file_exists(DIR_APPLICATION . '/' . DIRNAME_IMAGES . '/' . $filename . '.png')) {
+                $src = REL_DIR_APPLICATION . '/' . DIRNAME_IMAGES . '/' . $filename . '.png';
+            } else {
+                $src = ASSETS_URL_IMAGES . '/' . $filename . '.svg';
+            }
         }
 
-        return '<img id="ccm-logo" src="' . $src . '" alt="' . $alt . '" title="' . $alt . '" />';
+        return '<img id="ccm-logo" src="' . $src . '" alt="' . $alt . '" title="' . $alt . '" height="23" width="23">';
```

Comment:
I think the actual width/height was removed for whitelabel reasons, not sure though